src/main/groovy/io/seqera/wave/service/logs/BuildLogService.groovy

```diff
@@ -28,6 +28,9 @@ import io.micronaut.http.server.types.files.StreamedFile
  */
 interface BuildLogService {
 
+    String CONDA_LOCK_START = "conda_lock_start"
+    String CONDA_LOCK_END = "conda_lock_end"
```

Comment:
```suggestion
    String CONDA_LOCK_END = ">>>>>>> CONDA_LOCK_END"
```

Git marker style

Reply:
also it would be better to move in the implementation with `private static final`

Reply:
done